benchmarks/test_colander.py

```diff
@@ -0,0 +1,55 @@
+import pkg_resources
+import colander as c
+
+
+class TestColander:
+    package = "colander"
+    version = pkg_resources.get_distribution(package).version
+
+    def __init__(self, allow_extra):
+        class LocationSchema(c.MappingSchema):
+            latitude = c.SchemaNode(c.Float(), missing=None)
+            longitude = c.SchemaNode(c.Float(), missing=None)
+
+        class SkillSchema(c.MappingSchema):
+            subject = c.SchemaNode(c.Str())
+            subject_id = c.SchemaNode(c.Int())
+            category = c.SchemaNode(c.Str())
+            qual_level = c.SchemaNode(c.Str())
+            qual_level_id = c.SchemaNode(c.Int())
+            qual_level_ranking = c.SchemaNode(c.Float(), missing=0)
+
+        class SkillsSchema(c.SequenceSchema):
+            skill = SkillSchema()
+
+        class Model(c.MappingSchema):
+            id = c.SchemaNode(c.Int())
+            client_name = c.SchemaNode(c.Str(), validator=c.Length(max=255))
+            sort_index = c.SchemaNode(c.Float())
+            # client_email = fields.Email()
+            client_phone = c.SchemaNode(
+                c.Str(), validator=c.Length(max=255), missing=None
+            )
+            location = LocationSchema()
+            contractor = c.SchemaNode(c.Int(), validator=c.Range(min=0), missing=None)
+            upstream_http_referrer = c.SchemaNode(
+                c.Str(), validator=c.Length(max=1023), missing=None
+            )
+            grecaptcha_response = c.SchemaNode(
+                c.Str(), validator=c.Length(min=20, max=1000)
+            )
+            last_updated = c.SchemaNode(
+                c.DateTime(format="%Y-%m-%dT%H:%M:%S"), missing=None
```

Comment:
see #1568 - in summary we need to modify the benchmarks but all libraries should be able to accept ints as unix time stamps as well as strings and strings should work with timezones like `fromisoformat()`.

Reply:
internally `colander` use legacy `iso8601` lib, that not fully follow ISO format, so this will be required fully custom DateTime field implementation

Reply:
maybe simplest to wait until this issues raised in #1568 are resolved, then complete this.

I know the current situation is annoying and I agree completely custom datetime parsing doesn't make sense, but I don't see another way until the problems in pydantic are resolved in v2.